README.md

```diff
@@ -10,41 +10,79 @@
 
 # Temporary Note
 
-This project is not ready for deployment and the documentation may be incorrect.
+  Changes are made on a daily basis to documentation and files. Documentation may not be accurate due to the rapid changes that are occuring. In its current state, this repo is not ready for deployment.
 
 # Welcome!
 
 This is incoming v2 front-end application for [OperationCode](https://operationcode.org). We highly recommend [joining our organization](https://operationcode.org/join) to receive an invite to our Slack team. From there, you'll want to join the `#oc-projects` channel. You can get help from multiple professional developers, including people who have worked on the application since day 1!
 
-Before contributing, please review our [GitHub Workflow]().
+Before contributing, please review our [Contributing Guide](CONTRIBUTING.md)
 
-## Maintainers
+# Maintainers
 
 For information about the maintainers of the project, check out [MAINTAINERS.md](MAINTAINERS.md).
 
-## Quick Start
-
+# Quick Start
 If you're unsure of how to start this app or code for it, don't worry! You're our target audience!
 Please read our [Contributing Guide](CONTRIBUTING.md) to learn everything you need to be able to ask the right questions on our Slack team.
 
 If you're no stranger to React applications and the JavaScript ecosystem:
 
-Version Assumptions:
-[Click to see Node version](https://github.com/OperationCode/front-end/blob/master/.nvmrc)
-yarn@1.7.0
+Recommended Versions of tools used within the repo:
+Node @ 8.11.3 LTS
+Yarn@1.7.0 or greater
+Git @2.17.1 or greater
+
+## Github workflow
```

Comment:
I don't think `##GitHub Workflow` should be within the README. Quick start is meant for experience open source contributors who are aware of the need to fork. For quick start, let's assume they've forked and clone the repo. It's also a good idea to describe this assumption just above the tool version assumptions.